tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+#dist handled by poetry
+skipsdist=true
+envlist = py39
```

Comment:
I think testing on `py38` and `py39` should be sufficient, given that Vestapol is being used on the following platforms
* Our [Google Cloud Functions](https://console.cloud.google.com/functions/list?referrer=search&project=inq-warehouse) run on Python 3.9
* Our [Cloud Composer (Airflow) environment](https://console.cloud.google.com/composer/environments/detail/us-central1/inq-warehouse-airflow-test/configuration?referrer=search&project=inq-warehouse) is on image version `composer-2.0.0-airflow-2.1.4`, [which uses Python 3.8.12](https://cloud.google.com/composer/docs/concepts/versioning/composer-versions)